lib/ability.rb

```diff
@@ -28,7 +28,7 @@ def initialize(user, resource, controller)
         can :manage, [AccountPriceGroupMember, UserPriceGroupMember]
       end
 
-      if resource.global? && (user.user_roles.map(&:role) & UserRole.facility_roles).any?
+      if resource.global? && (user.user_roles.map(&:role) & UserRole.facility_roles).any? && resource.admin_editable?
```

Comment:
Can we extract this conditional into a method with a name specifying what it's looking at?